packages/@okta/vuepress-theme-prose/global-components/ApiAuthMethodWarning.vue

```diff
@@ -0,0 +1,11 @@
+<template>
+  <blockquote>
+    <p><strong>Note</strong>: Some of the curl code examples on this page include SSWS API token authentication. However, Okta recommends using scoped OAuth 2.0 and OIDC access tokens to authenticate with Okta management APIs. OAuth 2.0 and OIDC access tokens provide fine-grain control over the bearer's actions on specific endpoints compared to SSWS tokens. See Okta API <a href="/docs/reference/core-okta-api/#authentication">authentication</a> methods.</p>
```

Comment:
```suggestion
    <p><strong>Note</strong>: Some of the curl code examples on this page include SSWS API token authentication. However, Okta recommends using scoped OAuth 2.0 and OIDC access tokens to authenticate with Okta management APIs. OAuth 2.0 and OIDC access tokens provide fine-grain control over the bearer's actions on specific endpoints. See Okta API <a href="/docs/reference/core-okta-api/#authentication">authentication methods</a>.</p>
```

Reply:
Suggestion to remove "compared to SSWS tokens". Seemed unnecessary. Also expanded the link to include "methods", mainly to adhere to our style.

Reply:
Thanks for the suggestion! I'll apply these and confirm with Kim Maida and Jeff Taylor